scripts/release/mule/Makefile.mule

```diff
@@ -4,7 +4,7 @@ PKG_DIR = $(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH)
 
 .PHONY: ci-clean ci-setup ci-build
 
-ci-clean:
+ci-clean: clean
```

Comment:
What does clean do? Is it like `cleanWs()` which cleans the Workspace?

Reply:
@bricerisingalgorand a little help? I think it deletes temp files, but i'm not 100% sure.

Reply:
https://github.com/algorand/go-algorand/blob/master/Makefile#L252

Reply:
That tries to delete all build/generated files in the current repo